tests/e2e/default-mobile/reports/messages.wdio-spec.js

```diff
@@ -0,0 +1,64 @@
+const utils = require('@utils');
+const placeFactory = require('@factories/cht/contacts/place');
+const userFactory = require('@factories/cht/users/users');
+const personFactory = require('@factories/cht/contacts/person');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const commonElements = require('@page-objects/default/common/common.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const pregnancyFactory = require('@factories/cht/reports/pregnancy');
+
+describe('Report', () => {
```

Comment:
```suggestion
describe('Report - Send message action', () => {
```
The file can be `send-message.wdio-spec.js` 

I've removed myself as an approver and added @tatilepizs; she's been reorganizing the e2e, so she might have additional feedback

Reply:
Done